java-squid/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -84,6 +84,7 @@ public boolean equals(Object obj) {
     final ProgramPoint programPoint;
     @Nullable
     final ProgramState programState;
+    public boolean exitPath = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make exitPath a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)